report/5.Solution/DockerBuild.tex

```diff
@@ -0,0 +1,113 @@
+\section{Build process}\label{sec:buildprocess}
+In previous semesters, the build process has been straightforward, as it has simply been a matter of using the tool included with the IDE of choice for the chosen language.
```

Comment:
Måske ikke "previous semesters".. skal/må vi reffe det? Ellers så noget med "in other projects" eller lignende?